src/renderer/hooks/customHooks.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from 'react';
+
+function useCustomHook(
+  handleContinue: () => Promise<void>,
+  condition: boolean
```

Comment:
Would love a more descriptive name on this - maybe keypressEnabled or something similar?

Reply:
Went with `useKeypress` in [2cfe8cd](https://github.com/chloebrett/mlvet/pull/175/commits/2cfe8cdf2d2f5cadb584f5f938d5cbaa3a43f3f3)

Reply:
I think @emmapainter was talking about renaming the `conditional`. I like that idea. `isKeypressEnabled` 😄

Reply:
Ahh i see, I updated it here: [e947eb3](https://github.com/chloebrett/mlvet/pull/175/commits/e947eb3ff17d018bc491c0d2526cf921bac915aa)